internal/pubsub/samples/sub/main.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2018 The Go Cloud Authors
```

Comment:
`sub` seems to overlap heavily with `pub`... WDYT about combining them? `gocloudpubsub pub mytopic`, `gocloudpubsub sub mytopic`....

Reply:
It's a fair point about overlapping code, but I'm not liking the verbosity and stuttering.

How about we call it `msgs`. Then we can write for example:
```
msgs sub $subscription
echo "the eagle has landed" | msgs pub $topic
```

Reply:
Or maybe `msgs send $topic` and `msgs recv $subscription`.

Reply:
It turns out that `msgs` is the name of an existing unix command. Other possibilities include `msg` and `gcmsg`.

Reply:
Done.